docs/docs/configuration/alpha_config.md

```diff
@@ -296,6 +296,7 @@ Provider holds all configuration for a single provider
 | `loginURL` | _string_ | LoginURL is the authentication endpoint |
 | `redeemURL` | _string_ | RedeemURL is the token redemption endpoint |
 | `profileURL` | _string_ | ProfileURL is the profile access endpoint |
+| `introspectURL` | _string_ | IntrospectURL is the claims and permissions access Endpoint (OIDC) |
```

Comment:
oh, checked okta, keycloak and HSDP iam's well known discovery endpoints and i can see the introspection_endpoint so i added it in OIDC provider...then we can move this enrich session functionality to Hsdp iam provider i guess

Reply:
It may well be one of those "unofficial" standards then ¯\_(ツ)_/¯

It might be worth opening an issue on the upstream to discuss this addition and see if it would be included in the default.

Im quite sure there is no agreed format for the introspect response (not even sure it needs to be json). But as we are just passing it through that should not be an issue.